build/linux/installer/conf/fluent-bit-geneva-logs_infra.conf

```diff
@@ -25,3 +25,16 @@
     Retry_Limit                     10
     TelemetryPushIntervalSeconds    300
     Match                           *_<INFRA_NAMESPACE>_*
+
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               geneva.container.log.*
```

Comment:
should be geneva.container.log.infra.*


also move this between input and output